documentation/src/main/asciidoc/topics/proc_enabling_jmx_port.adoc

```diff
@@ -0,0 +1,23 @@
+[id='enabling-jmx-port_{context}']
+= Enabling JMX remote ports
+Provide unique remote JMX ports to expose {brandname} MBeans through connections in JMXServiceURL format.
+
+// Server Guide
+ifdef::remote_caches[]
+[NOTE]
+====
+{brandname} Server does not expose JMX remotely by using the single port endpoint. If you want to remotely access the {brandname} Server through JMX, you must enable a remote port.
```

Comment:
Each sentence should start on its own line
```suggestion
{brandname} Server does not expose JMX remotely by using the single port endpoint.
If you want to remotely access the {brandname} Server through JMX, you must enable a remote port.
```

Reply:
Thanks, @dvagnero .

I updated the content based on your feedback.